### DISCUSSION TITLE
Contributing new examples in examples directory.

### DISCUSSION THREAD

**Discussion by manupatel007** [python-telegram-bot/python-telegram-bot#2673] [Q&A] [answered]
Just wanted to ask if this repository accepts new examples of telegram bot created with python-telegram-bot.

**Comment by Bibo-Joshi** [comment, marked as answer]
Hi. This depends n the example. The examples provided by PTB are mainly intended to showcase the features of PTB itself. In constrast to that, this [wiki page](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bots-built-with-PTB) gathers bots written with PTB by the community.
